src/main/java/connor/Connor.java

```diff
@@ -0,0 +1,113 @@
+package connor;
+
+import connor.command.ByeCommand;
+import connor.exception.InvalidTaskFileException;
+import connor.task.TaskList;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.Scanner;
+
+/**
+ * Main class for Connor program to start running.
+ *
+ * @author jaysmyname
+ * @version 1.0.0
+ */
+public class Connor {
+    public static final String CURRENT_VERSION = "Version 1.0.0";
+    public static final String LINE = "_".repeat(66);
+    public static final String INDENT = " ".repeat(4);
+    public static final String INPUT_HERE = ">>> ";

```

Comment:
I like that all your static variables are all in full caps, which is the trend among all your files. Good job!